evm-template/runtime/src/configs/xcm_config.rs

```diff
@@ -232,50 +223,11 @@ parameter_types! {
     pub TreasuryAccount: AccountId = Treasury::account_id();
 }
 
-pub struct XcmConfig;
-impl xcm_executor::Config for XcmConfig {
-    type Aliasers = Nothing;
-    type AssetClaims = PolkadotXcm;
-    type AssetExchanger = ();
-    type AssetLocker = ();
-    // How to withdraw and deposit an asset.
-    type AssetTransactor = AssetTransactors;
-    type AssetTrap = PolkadotXcm;
-    type Barrier = Barrier;
-    type CallDispatcher = RuntimeCall;
-    /// When changing this config, keep in mind, that you should collect fees.
-    type FeeManager = XcmFeeManagerFromComponents<
-        IsChildSystemParachain<primitives::Id>,
-        XcmFeeToAccount<Self::AssetTransactor, AccountId, TreasuryAccount>,
-    >;
-    type HrmpChannelAcceptedHandler = ();
-    type HrmpChannelClosingHandler = ();
-    type HrmpNewChannelOpenRequestHandler = ();
-    /// Please, keep these two configs (`IsReserve` and `IsTeleporter`) mutually exclusive.
-    /// The IsReserve type must be set to specify which <MultiAsset, MultiLocation> pair we trust to deposit reserve assets on our chain. We can also use the unit type () to block ReserveAssetDeposited instructions.
-    /// The IsTeleporter type must be set to specify which <MultiAsset, MultiLocation> pair we trust to teleport assets to our chain. We can also use the unit type () to block ReceiveTeleportedAssets instruction.
-    type IsReserve = Reserves;
-    type IsTeleporter = ();
-    type MaxAssetsIntoHolding = MaxAssetsIntoHolding;
-    type MessageExporter = ();
-    type OriginConverter = XcmOriginToTransactDispatchOrigin;
-    type PalletInstancesInfo = AllPalletsWithSystem;
-    type ResponseHandler = PolkadotXcm;
-    type RuntimeCall = RuntimeCall;
-    type SafeCallFilter = Everything;
-    type SubscriptionService = PolkadotXcm;
-    type Trader = (
-        UsingComponents<WeightToFee, BalancesPalletLocation, AccountId, Balances, ()>,
-        xcm_primitives::FirstAssetTrader<AssetType, AssetManager, XcmFeesToAccount>,
-    );
-    type TransactionalProcessor = FrameTransactionalProcessor;
-    type UniversalAliases = Nothing;
-    // Teleporting is disabled.
-    type UniversalLocation = UniversalLocation;
-    type Weigher = FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
-    type XcmRecorder = PolkadotXcm;
-    type XcmSender = XcmRouter;
-}
+/// When changing this config, keep in mind, that you should collect fees.
```

Comment:
If this is only relevant if someone wants to change the default we provide(correct me if that's not the case), then I think this is clearer. 
Also should the comment at least point to something that helps you know how/what to change and the consequences?
```suggestion
/// If you change this config, keep in mind that you should define how you collect fees.
```